dashboard/v1.1/src/pages/Monitoring/Delay.tsx

```diff
@@ -42,7 +44,16 @@ const Delay: FC<DelayProps> = ({ delay }) => {
       }
     }
   };
-  return <Chart series={series} options={options} height="300" />;
+  return (
+    <>
+      {!delay.length && (
+        <Alert severity="error">
+          No data found for {`<monitoring/response-delay>`}
```

Comment:
I think we can keep a single `No data found` across all the graphs. The ideal way for this would be to create a custom graph component that implements the apex chart. This component takes in all the props and based on that make a graph. If the data in the property is empty, simply render `No data found`.

Reply:
Please note: The entire graph component should be disabled and only the `No data found` will be shown.

Reply:
Sure, that sounds good!